ts/features/design-system/components/DSSpacerViewerBox.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from "react";
+import { View, StyleSheet, Text } from "react-native";
+import {
+  Spacer,
+  SpacerOrientation
+} from "../../../components/core/spacer/Spacer";
+import { IOColors } from "../../../components/core/variables/IOColors";
+import type { IOSpacer } from "../../../components/core/variables/IOSpacing";
+
+const styles = StyleSheet.create({
+  spacerWrapper: {
+    backgroundColor: IOColors.greyLight
+  },
+  componentLabel: {
+    fontSize: 9,
+    color: IOColors.bluegrey
+  }
+});
+
+type DSSpacerViewerBoxProps = {
+  size: IOSpacer;
+  orientation?: SpacerOrientation;
+};
+
+export const DSSpacerViewerBox = ({
```

Comment:
what is the meaning of the prefix `DS`?  are there others `SpaceViewerBox` in the project?

You can use "named imports," as described [here](https://www.typescriptlang.org/docs/handbook/modules.html#import-the-entire-module-into-a-single-variable-and-use-it-to-access-the-module-exports), to place your components in a specific namespace.

Reply:
We already use `named imports` to make a distinction between RN's `View` and NativeBase's `View` components. (that's why you could come across `NBView` at some point).

`DS` means `Design System`. `DSSpaceViewerBox` is a specific component designed to show spacing values in the Design System page. You shouldn't include it in other pages except that one.